frontend/src/app/main/ui/workspace/colorpicker/ramp.cljs

```diff
@@ -63,38 +64,52 @@
            :h h :s s :v v
            :r r :g g :b b)))
 
-(mf/defc ramp-selector
+(mf/defc ramp-selector*
   [{:keys [color disable-opacity on-change on-start-drag on-finish-drag]}]
-  (let [internal-color
+  (let [internal-color*
         (mf/use-state #(enrich-color-map color))
 
-        {:keys [h s v hex alpha]} @internal-color
+        internal-color
+        (deref internal-color*)
+
+        h     (get internal-color :h)
+        s     (get internal-color :s)
+        v     (get internal-color :v)
+        hex   (get internal-color :hex)
+        alpha (get internal-color :alpha)
+
+        bullet-color
+        (mf/with-memo [hex alpha]
```

Comment:
Store the value in a memoed variable, to avoid creating the map in each rerender. So the gc has less work.